components/expression/expression-panel.html

```diff
@@ -31,7 +43,7 @@
       </tr>
       <tr>
         <td>Condition Labels</td>
-        <td><input type="checkbox" id="label-conditions" title="show experiment condition (condition) labels" checked></td>
```

Comment:
It does not mean that the labels should be unchecked by default. You may want to resolve the height error using other method.